infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -71,6 +67,14 @@ public boolean recoverCluster(ServerContext serverContext, Layout recoveryLayout
             if (corfuRuntime.getLayoutView().getLayout().equals(recoveryLayout)) {
                 log.info("Layout Recovered = {}", recoveryLayout);
             }
+
+            //TODO: Since sequencer reset is moved after paxos. Make sure the runtime has the latest
```

Comment:
Need to handle the case when the !corfuRuntime.getLayoutView().getLayout().equals(recoveryLayout).

At least  log something.